WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -1458,6 +1460,34 @@ private extension ProductFormViewController {
     }
 }
 
+// MARK: Action - Show Custom Fields
+private extension ProductFormViewController {
+    func showCustomFields() {
+        guard let product = product as? EditableProductModel else {
+            return
+        }
+
+        let customFields = product.product.customFields.map {
+            CustomFieldViewModel(metadata: $0)
+        }
+
+        let customFieldsView = UIHostingController(
+            rootView: CustomFieldsListView(
+                isEditable: true,
+                viewModel: CustomFieldsListViewModel(customFields: customFields),
+                onBackButtonTapped: { [weak self] in
+                    // Restore the hidden navigation bar
+                    self?.navigationController?.setNavigationBarHidden(false, animated: false)
+            })
+        )
+
+        // Hide the navigation bar as `CustomFieldsListView` will create its own toolbar.
+        navigationController?.setNavigationBarHidden(true, animated: false)
+        navigationController?.pushViewController(customFieldsView, animated: false)
```

Comment:
Same comment as above about keeping the animation for the push transition.

Reply:
https://github.com/woocommerce/woocommerce-ios/pull/14068/commits/d513b5400bffcd894a11166547d417702fc34aad